File: Application/src/com/hhp227/application/VerinfoActivity.java
package com.hhp227.application;

import android.app.ActionBar;
import android.app.Activity;
import android.os.Bundle;

public class VerinfoActivity extends Activity {
	
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_verinfo);
		ActionBar actionBar = getActionBar();
		actionBar.setDisplayHomeAsUpEnabled(true);
        actionBar.setDisplayShowHomeEnabled(false);
	}
	
	/**
	 * 액션바에 뒤로가기 버튼을 누르면 나타나는 동작
	 */
	public boolean onOptionsItemSelected(android.view.MenuItem item) {
        switch(item.getItemId()) {
            case android.R.id.home :
                finish();
                return true;
        }
        return super.onOptionsItemSelected(item);
    }

}

File: app/src/main/java/com/hhp227/application/activity/VerInfoActivity.kt
package com.hhp227.application.activity

import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import com.hhp227.application.BuildConfig
import com.hhp227.application.databinding.ActivityVerinfoBinding

class VerInfoActivity : AppCompatActivity() {
    private lateinit var binding: ActivityVerinfoBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityVerinfoBinding.inflate(layoutInflater)
        binding.tvVerName.text = BuildConfig.VERSION_NAME

        setContentView(binding.root)
        setSupportActionBar(binding.toolbar)
        supportActionBar?.run { setDisplayHomeAsUpEnabled(true) }
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean = when (item.itemId) {
        android.R.id.home -> {
            onBackPressed()
            true
        }
        else -> super.onOptionsItemSelected(item)
    }
}